### PR TITLE
Apply example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Apply example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: conf/htconfig.sample.php [hunk 1]
```diff
@@ -66,11 +66,11 @@
 
 App::$config['system']['register_policy'] = REGISTER_OPEN;
 App::$config['system']['register_text'] = '';
-App::$config['system']['admin_email'] = '';
+App::$config['system']['admin_email'] = '__EMAIL__';
 
 // Location of PHP command line processor
 
-App::$config['system']['php_path'] = 'php';
+App::$config['system']['php_path'] = 'php__PHPVERSION__';
```
Comment (ericgaspar) [inline conf/htconfig.sample.php:73]:
```suggestion
App::$config['system']['php_path'] = '/usr/bin/php__PHPVERSION__';
```
We may need PHP path here...

**Comment by yalh76** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**